### PR TITLE
Cleanup Model Tools

### PR DESCRIPTION
**Pull request by ctaylo37**

### CODE REVIEW

**Comment by ctaylo37** [comment]
This fixes #321 not 312.

File: src/mmw/js/src/modeling/templates/controls/precipitation.html [hunk 1]
```diff
@@ -1,4 +1,6 @@
+<div class="btn-sm btn-primary">
 <label>
-    Rain Level
+    Precip
```
Comment (lliss) [inline src/mmw/js/src/modeling/templates/controls/precipitation.html]: Do we really not want to use the full word here?

**Comment by rajadain** [comment]
Screenshot from Firefox (Retina):

screen shot 2015-06-15 at 1 04 27 pm

from Firefox:

screen shot 2015-06-15 at 1 09 10 pm

from Chrome:

screen shot 2015-06-15 at 1 05 07 pm

from Safari:

screen shot 2015-06-15 at 1 06 43 pm

The Precip box is lower than the other three, and the label font is bolder.

Regarding **Precip** vs **Rain Level**, here's a screenshot on an iPad in portrait resolution (taken using Firefox's responsive design mode):

screen shot 2015-06-15 at 13 12 37

There's plenty of room to use the full word, and we can use **Precipitation**.

This work will also have to be rebased on top of `develop` since #390 has caused merge conflicts here.

**Comment by kdeloach** [comment]
Can we close this in favor of #438? We can address minor styling fixups in a future PR.

**Comment by caseypt** [comment]
Rebased in #438

**Pull request closed without merging**